modules/ting/src/TingObject.php

```diff
@@ -0,0 +1,871 @@
+<?php
+/**
+ * @file
+ * The TingPseudoObject class.
+ */
+
+namespace Ting;
+
+/**
+ * Value Object implementation of a TingObjectInterface.
+ *
+ * Can be used to manually construct a TingObjectInterface compatible object.
+ *
+ * @package Ting
+ */
+class TingObject implements TingObjectInterface {
+
+  public $abstract;
+  public $age;
+  public $audience;
+  public $classifications;
+  public $contributors;
+  public $creatorsFormatSurnameFirst;
+  public $creatorsFormatDefault;
+  public $description;
+  public $extent;
+  public $format;
+  public $genere;
+  public $id;
+  public $isbn;
+  public $isLocal;
+  public $isPartOf;
+  public $language;
+  public $materialSource;
+  public $musician;
+  public $online;
+  public $onlineUrl;
+  public $ownerId;
+  public $pegi;
+  public $publisher;
+  public $referenced;
+  public $relations = [];
+  public $replacedBy;
+  public $replaces;
+  public $rights;
+  public $seriesDescription;
+  public $seriesTitles;
+  public $shortTitle;
+  public $source;
+  public $sourceId;
+  public $spatial;
+  public $spoken;
+  public $subjects;
+  public $subTitles;
+  public $title;
+  public $tracks;
+  public $type;
+  public $uRI;
+  public $version;
+  public $year;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function isLocal() {
+    return $this->isLocal;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getRelations() {
+    return $this->relations;
+  }
+
+  // TODO BBS-SAL BBS-153: Temporary fix to allow the pseudo object to be accessed via
```

Comment:
Is the SAL port not complete?

Reply:
This may be related to the lacking migration of ting_new_materials. What do you say @danquah?